docs/source/user_guide/memory.rst

```diff
@@ -198,3 +198,60 @@ Be sure to do this before any other CuPy operations.
 
    # Disable memory pool for pinned memory (CPU).
    cupy.cuda.set_pinned_memory_allocator(None)
+
+
+System allocated memory (SAM) support (**experimental!**)
+.........................................................
+
+It is possible to make both NumPy and CuPy use and share system allocated memory on Heterogeneous Memory Management
+(HMM) enabled systems or the NVIDIA Grace Hopper Superchip. To activate this capability, currently you need to:
+
+1. Build CuPy from source and install `numpy_allocator <https://github.com/inaccel/numpy-allocator/>`_
```

Comment:
The doc was written during development, now that we're upstreaming it we can simplify it 🙂 
```suggestion
1. Install `numpy_allocator <https://github.com/inaccel/numpy-allocator/>`_
```